src/codemagic/tools/app_store_connect/action_groups/profiles_action_group.py

```diff
@@ -0,0 +1,147 @@
+from __future__ import annotations
+
+import time
+from abc import ABCMeta
+from typing import List
+from typing import Optional
+from typing import Sequence
+
+from codemagic import cli
+from codemagic.apple.resources import Profile
+from codemagic.apple.resources import ProfileState
+from codemagic.apple.resources import ProfileType
+from codemagic.apple.resources import ResourceId
+
+from ..abstract_base_action import AbstractBaseAction
+from ..action_group import AppStoreConnectActionGroup
+from ..arguments import BundleIdArgument
+from ..arguments import CertificateArgument
+from ..arguments import CommonArgument
+from ..arguments import ProfileArgument
+from ..errors import AppStoreConnectError
+
+
+class ProfilesActionGroup(AbstractBaseAction, metaclass=ABCMeta):
+    @cli.action(
+        "create",
+        BundleIdArgument.BUNDLE_ID_RESOURCE_ID,
+        CertificateArgument.CERTIFICATE_RESOURCE_IDS,
+        ProfileArgument.DEVICE_RESOURCE_IDS,
+        ProfileArgument.PROFILE_TYPE,
+        ProfileArgument.PROFILE_NAME,
+        CommonArgument.SAVE,
+        action_group=AppStoreConnectActionGroup.PROFILES,
+        deprecation_info=cli.ActionDeprecationInfo("create-profile", "0.49.0"),
+    )
+    def create_profile(
```

Comment:
Used to be defined in https://github.com/codemagic-ci-cd/cli-tools/blob/3670fa624997f17cb437bc88198f7bdee3594f35/src/codemagic/tools/app_store_connect.py#L666-L715